### PR TITLE
feat: Creates cc bcc input component story for reply box

### PR DESCRIPTION
**Pull request by nithindavid**
image
image

### CODE REVIEW

File: app/javascript/dashboard/components/widgets/conversation/stories/ReplyEmailHead.stories.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import AddReminder from '../ReplyEmailHead';
```
Comment (muhsin-k) [inline app/javascript/dashboard/components/widgets/conversation/stories/ReplyEmailHead.stories.js:1]:
```suggestion
import ReplyEmailHead from '../ReplyEmailHead';
```

File: app/javascript/dashboard/components/widgets/conversation/ReplyEmailHead.vue [hunk 2]
```diff
@@ -0,0 +1,119 @@
+<template>
+  <div>
+    <div class="input-group-wrap">
+      <div class="input-group small" :class="{ error: $v.ccEmails.$error }">
+        <label class=" input-group-label">
```
Comment (muhsin-k) [inline app/javascript/dashboard/components/widgets/conversation/ReplyEmailHead.vue:5]: Space before classname

File: app/javascript/dashboard/components/widgets/conversation/ReplyEmailHead.vue [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+          {{ $t('CONVERSATION.REPLYBOX.EMAIL_HEAD.ADD_BCC') }}
+        </woot-button>
+      </div>
+      <span v-if="$v.ccEmails.$error" class="message">
+        {{ $t('CONVERSATION.REPLYBOX.EMAIL_HEAD.CC.ERROR') }}
+      </span>
+    </div>
+    <div v-if="showBcc" class="input-group-wrap">
+      <div class="input-group small" :class="{ error: $v.bccEmails.$error }">
+        <label class=" input-group-label">
```
Comment (muhsin-k) [inline app/javascript/dashboard/components/widgets/conversation/ReplyEmailHead.vue:32]: Space before classname

**Review by muhsin-k — CHANGES_REQUESTED** [review]
@nithindavid Nice work. Please see the inline comments

**Comment by nithindavid** [comment]
@muhsin-k thanks for the review,  I have made the changes.

File: app/javascript/dashboard/components/widgets/conversation/stories/ReplyEmailHead.stories.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      control: {
+        type: 'string',
+      },
+    },
+  },
+};
+
+const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { AddReminder },
```
Comment (muhsin-k) [inline app/javascript/dashboard/components/widgets/conversation/stories/ReplyEmailHead.stories.js:22]:
```suggestion
  components: { ReplyEmailHead },
```

File: app/javascript/dashboard/components/widgets/conversation/stories/ReplyEmailHead.stories.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  template:
+    '<add-reminder v-bind="$props" @add="onAdd" @click="onClick"></add-reminder>',
```
Comment (muhsin-k) [inline app/javascript/dashboard/components/widgets/conversation/stories/ReplyEmailHead.stories.js:24]:
```suggestion
    '<reply-email-head v-bind="$props" @add="onAdd" @click="onClick"></reply-email-head>',
```

File: app/javascript/dashboard/components/widgets/conversation/stories/ReplyEmailHead.stories.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import ReplyEmailHead from '../ReplyEmailHead';
+
+export default {
+  title: 'Components/ReplyBox/EmailHead',
+  component: AddReminder,
```
Comment (muhsin-k) [inline app/javascript/dashboard/components/widgets/conversation/stories/ReplyEmailHead.stories.js:5]:
```suggestion
  component: ReplyEmailHead,
```

**Review by muhsin-k — APPROVED** [review]

**Pull request merged**